src/cli/utils/sharepoint.go

```diff
@@ -66,6 +66,33 @@ func MakeSharePointOpts(cmd *cobra.Command) SharePointOpts {
 	}
 }
 
+func SharePointAllowedCategories() map[string]struct{} {
+	return map[string]struct{}{
+		flags.DataLibraries: {},
+		flags.DataLists:     {},
+		flags.DataPages:     {},
```

Comment:
I think you're carrying over old code here, but we do _not_ support backup of pages at this time, and should not display that category in our supported set.  Can you please ensure pages is not allowed (or displayed as supported) in the cli?

Reply:
yeah correct. I will remove 'pages' from here, so that even if we explicitely try to backup pages, we get:
`Error: pages is an unrecognized data type; either  libraries or lists`